wework/scripts/dev-mac-app.sh

```
@@ -243,8 +269,13 @@ if [ ! -x "$DWS_BINARY_PATH" ]; then
   echo "Error: DWS binary is not executable: $DWS_BINARY_PATH" >&2
   exit 1
 fi
-WEWORK_EXECUTOR_PROFILE=debug pnpm --dir electron prepare:package
-export WEWORK_COMPONENT_RESOURCES_ROOT="$WEWORK_DIR/electron/resources"
+node "$SCRIPT_DIR/prepare-dev-component-resources.mjs"
+export WEWORK_CORE_PLUGINS_SHA256="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)